canvasapi/canvas_object.py

```diff
@@ -77,5 +87,16 @@ def to_json(self):
         """
         Return the original JSON response from the API that was used to
         construct the object.
+
+        .. warning::
+            .. deprecated:: 0.14.0
```

Comment:
Deprecation is actually happening in 0.15.0, right?

Reply:
True! Fixed.